README.md

```diff
@@ -27,3 +30,23 @@ python mpas_xarray.py -f "globalStats*nc"
 
 This outputs a simple time-series plot of Time vs. Time to test functionality.
 
+ Example Usage for timeSeriesStatsAM fields:
+
+```
+from mpas_xarray import preprocess_mpas_timeSeriesStats, remove_repeated_time_index
+
+ds = xarray.open_mfdataset('am.mpas-cice*nc', preprocess=preprocess_mpas_timeSeriesStats)
+ds = remove_repeated_time_index(ds)
+```
+
+To test:
+
+```
+tar xzvf am.mpas-ciceShort.tgz
+python mpas_xarray.py -f "am.mpas-cice*nc" --istimeavg "true"
+```
+
+This plots a short time series of global average ice concentration, showing the correctly centered curve
+(derived using preprocess_mpas_timeSeriesStats) and the curve incorrectly shifted toward the end of the 
+time averaging period (derived using preprocess_mpas).
```

Comment:
We could include an image for the examples if you think this is helpful.  However, it is not necessary and may not be useful.  I could go either way on this.